pages/_app.tsx

```diff
@@ -8,6 +8,8 @@ import { VERCEL_ENV } from '@/next.constants.mjs';
 import type { AppProps } from 'next/app';
 
 import '@/styles/old/index.scss';
+// uncomment to unable tailwind (it's working)
```

Comment:
This comment is unnecessary. Please remove it 👀

Reply:
Sure ? ok ✅